src/main/java/io/vertx/ext/apex/core/impl/Utils.java

```diff
@@ -115,7 +116,7 @@ public static String readResourceToString(String resource) {
   }
 
   public static DateFormat createISODateTimeFormatter() {
-    DateFormat dtf = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz");
+    DateFormat dtf = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.US);
```

Comment:
I think there are no differences between en_US and en_GB regarding the date descriptions, most of the unix design was based initially in the US so the conventions of the C libraries are as well.
I think Locale.ENGLISH would work as well if you prefer that.